pkg/routers/api.go

```diff
@@ -124,6 +127,7 @@ func WithAPIRouter(opt *APIOptions, r chi.Router) {
 	}
 
 	r.Use(middleware.SetTime)
+	r.Use(opt.allowedContentTypesMiddleware)
```

Comment:
Why do we need to add `allowedContentTypesMiddleware` to `APIOptions`? Instead can we just do this here: 
```suggestion
	r.Use(chimw.AllowContentType("application/json"))
```